hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/parser/record/entity/EntityRecordItemListener.java

```diff
@@ -386,23 +386,27 @@ private void insertContractCall(long consensusTimestamp,
                                     ContractCallTransactionBody transactionBody,
                                     TransactionRecord transactionRecord) {
         if (entityProperties.getPersist().isContracts() && transactionRecord.hasContractCallResult()) {
-            byte[] functionParams = Utility.toBytes(transactionBody.getFunctionParameters());
-            long gasSupplied = transactionBody.getGas();
-            byte[] callResult = transactionRecord.getContractCallResult().toByteArray();
-            long gasUsed = transactionRecord.getContractCallResult().getGasUsed();
-            insertContractResults(consensusTimestamp, functionParams, gasSupplied, callResult, gasUsed);
+            ContractResult contractResult = new ContractResult();
```

Comment:
nit: could use a shared method so Contract creation is in one location
```suggestion
            private void insertContract(long consensusTimestamp, ByteString parameters,
                                       ContractFunctionResult contractFunctionResult, long gasLimit) {
        ContractResult contractResult = new ContractResult();
        contractResult.setConsensusTimestamp(consensusTimestamp);
        contractResult.setFunctionParameters(Utility.toBytes(parameters));
        contractResult.setFunctionResult(contractFunctionResult.toByteArray());
        contractResult.setGasLimit(gasLimit);
        contractResult.setGasUsed(contractFunctionResult.getGasUsed());
        entityListener.onContractResult(contractResult);
    }
```

Reply:
Okay